CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v0.12.0 - 2024/02/28
+
+### Changes
+
+- Added dotnet-isolated support to AssistantSkills
```

Comment:
Let's also add a `### Breaking changes` section for this release and mention the namespace changes.